player/http_routes.go

```diff
@@ -27,6 +27,7 @@ func InstallPlayerRoutes(r *mux.Router, p *Player) {
 
 	v4Router := r.PathPrefix("/api/v4").Subrouter()
 	v4Router.Path("/streams/free/{claim_name}/{claim_id}/{sd_hash}").HandlerFunc(playerHandler.Handle).Methods(http.MethodGet, http.MethodHead)
+	v4Router.Path("/status").HandlerFunc(playerHandler.Status).Methods(http.MethodGet, http.MethodPost)
```

Comment:
I would still suggest using one of the [industry-accepted urls](https://kubernetes.io/docs/reference/using-api/health-checks/) for the endpoint.